services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/application/ApplicationDataMapper.kt

```diff
@@ -0,0 +1,74 @@
+package fi.hel.haitaton.hanke.application
+
+import fi.hel.haitaton.hanke.COORDINATE_SYSTEM_URN
+import fi.hel.haitaton.hanke.allu.AlluCableReportApplicationData
+import fi.hel.haitaton.hanke.geometria.UnsupportedCoordinateSystemException
+import org.geojson.GeometryCollection
+
+object ApplicationDataMapper {
+
+    fun toAlluData(applicationData: CableReportApplicationData): AlluCableReportApplicationData =
+        AlluCableReportApplicationData(
+            applicationData.name,
+            applicationData.customerWithContacts.toAlluData("applicationData.customerWithContacts"),
+            getGeometry(applicationData),
+            applicationData.startTime
+                ?: throw AlluDataException("applicationData.startTime", "Can't be null"),
```

Comment:
The error string "Can't be null" is repeated at least three times. A constant could provide consistency and protection from typos.

Reply:
Yes, thank you. The exception is called with just two different error descriptions, so I created an enum for them.